src/Common/Auth/OneTimeAuth.php

```diff
@@ -330,7 +330,6 @@ public function processOnetime($token, $redirect_token): array
             }
         } catch (OneTimeAuthExpiredException $e) {
             $this->systemLogger->error("Failed " . $e->getMessage());
-            unset($auth);
```

Comment:
auth is set in line 323

Reply:
oh, i think i get it. auth will not be set here since that means the function it called threw an error/exception. correct?